types/commerce7__admin-ui/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+    "rules": {
+        "@definitelytyped/no-any-union": "off",
```

Comment:
Per @peterblazejewicz, handling these inline or adjusting the types to not require disabling the lint rules would be preferred.